controllers/submariner/gateway_resources.go

```diff
@@ -217,7 +217,7 @@ func newGatewayPodTemplate(cr *v1alpha1.Submariner, name string, podSelectorLabe
 			HostNetwork:                   true,
 			TerminationGracePeriodSeconds: pointer.Int64(1),
 			RestartPolicy:                 corev1.RestartPolicyAlways,
-			DNSPolicy:                     corev1.DNSClusterFirst,
+			DNSPolicy:                     corev1.DNSClusterFirstWithHostNet,
```

Comment:
Could you move this up a couple of lines so it’s immediately after `HostNetwork: true`? That way if we ever change the `HostNetwork` setting we’ll have a better chance of noticing the associated `DNSPolicy`.